Formula/primecount.rb

```diff
@@ -0,0 +1,37 @@
+class Primecount < Formula
+  desc "Fast prime counting function program and C/C++ library"
+  homepage "https://github.com/kimwalisch/primecount"
+  url "https://github.com/kimwalisch/primecount/archive/v7.3.tar.gz"
+  sha256 "471fe21461e42e5f28404e17ff840fb527b3ec4064853253ee22cf4a81656332"
+  license "BSD-2-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "libomp"
+  depends_on "primesieve"
+
+  def install
+    # In 2022 integer division is slow on most CPUs,
+    # hence by default we use libdivide instead.
+    use_libdivide = "ON"
+    use_div32 = "ON"
+
+    if OS.mac? && Hardware::CPU.arm?
+      # Apple Silicon CPUs have very fast integer division
+      use_libdivide = "OFF"
+      use_div32 = "OFF"
+    end
```

Comment:
Let's use the upstream defaults instead.

Reply:
OK, I have removed that.